RevenueCatUI/PaywallView.swift

```diff
@@ -170,29 +170,42 @@ private extension PaywallViewMode {
 struct PaywallView_Previews: PreviewProvider {
 
     static var previews: some View {
-        let offering = TestData.offeringWithNoIntroOffer
-
-        if let paywall = offering.paywall {
-            ForEach(PaywallViewMode.allCases, id: \.self) { mode in
+        ForEach(Self.offerings, id: \.self) { offering in
+            ForEach(Self.modes, id: \.self) { mode in
                 PaywallView(
                     offering: offering,
-                    paywall: paywall,
+                    paywall: offering.paywall!,
                     mode: mode,
                     introEligibility: Self.introEligibility,
                     purchaseHandler: Self.purchaseHandler
                 )
                 .previewLayout(mode.layout)
+                .previewDisplayName("\(offering.identifier)-\(mode)")
```

Comment:
This provides a nice title to the previews:
<img width="819" alt="Screenshot 2023-07-20 at 18 30 14" src="https://github.com/RevenueCat/purchases-ios/assets/685609/24d932ce-8647-454b-b003-fcaa0b6caec1">